GameData/Kerbalism/System/Misc.cfg

```diff
@@ -96,3 +96,18 @@
 // ============================================================================
 
 @PART[kerbalism-container-*]:FINAL { !ContainerVolume = _ }
+
+
+// ============================================================================
+// Increase all crewable parts' EC because ECLSS uses EC constantly,
+// you wanna survive in the Apollo LM, right?
+// ============================================================================
+
+@PART[*]:HAS[#CrewCapacity[>0],@RESOURCE[ElectricCharge]]:NEEDS[FeatureHabitat]:AFTER[zzzKerbalism]
```

Comment:
change NEEDS[FeatureHabitat] to NEEDS[ProfileDefault]

Also in the Tags section change FOR[Kerbalism] to FOR[zzzKerbalism] as in the post on Discord